src/modules/cubedos-time_server-messages.adb

```diff
@@ -197,14 +205,18 @@ package body CubedOS.Time_Server.Messages
          -- a tick request to itself, that logic error should perhaps be logged somehow. It
          -- would be nice to statically show that situation can never happen.
          --
+         -- TODO: This needs to be updated to reflect having domains.
+         -- Before that can happen though, we need to figure out domains.
          if Incoming_Message.Sender_Address.Module_ID in Restricted_Module_ID then
```

Comment:
I'm not sure I understand the comment here. Why not just check against Name_Resolver.Time_Server? The name resolver has the time server's full address including domain ID and module ID information.